### PR TITLE
Add description for "TiDB Performance Challenge Program"

### PR DESCRIPTION
**Pull request by Soline324**
@zz-jason PTAL

### CODE REVIEW

File: challenge-Programs/performance-challenge-cn.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+### TiDB Performance Challenge Program
+
+TiDB 产品的每一次微小进步都离不开社区小伙伴的支持和帮助，我们很欣喜地看到：越来越多的小伙伴参与到 TiDB 开源社区的建设当中，越来越多的小伙伴在 TiDB 开源社区用自己的方式表达着对于开源的热情和对于技术的追求，TiDB 也在社区小伙伴的推动下，不断地刷新着过去的成绩。
+
+我们感谢并始终相信开源社区的力量，为了让 TiDB 产品在稳定性、性能、易用性等方面更上一层楼，我们决定推出一个中长期的挑战计划 —— TiDB Challenge Program，将一些 Issue 开放出来，供社区小伙伴共同探讨，共同解决，每一个 Issue 都将对应一定的积分，参赛选手可在每个赛季结束后用获得的积分兑换社区大礼。
+
+TiDB Performance Challenge Program 作为 TiDB Challenge Program 系列的第一赛季，将聚焦 Performance Improvement，于 2019 年 11 月 4 日正式开启，2020 年 2 月 4 日结束，持续 3 个月。
+
+欢迎大家加入 [TiDB Community Slack Workspace](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE)，参赛过程中遇到任何问题都可以直接通过 **#performance-challenge-program channel** 与我们取得联系。
```
Comment (gingerkidney) [inline challenge-Programs/performance-challenge-cn.md]:
```suggestion
欢迎大家加入 [TiDB Community Slack Workspace](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE)，参赛过程中遇到任何问题都可以直接通过 **#performance-challenge-program** channel 与我们取得联系。
```

File: challenge-Programs/performance-challenge-cn.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+如果你决定认领某一个 issue，请先在这个 Issue 中 回复 **“/pick-up-challenge”**， 后台将自动判断你所拥有的积分是否具备挑战此 Issue 的资格，积分满足要求即可开始挑战，积分不满足要求，需按照系统提示获得满足挑战要求的积分。
+
+**需要特别提醒的是：**
+
+- 每个参赛主体（含个人及团队）参与 TiDB 性能挑战赛的**初始积分为 0**，需要先完成 “Easy” 的 Issue 将积分积累至 400 分以上（含 400 分），才有资格挑战难度为“Medium”和“Hard”的题目
+- 每个参赛主体一次只能领取一个任务
+
+#### 第三步：实现代码
+
+在实现代码的过程中如果遇到问题，可以通过 **#performance-challenge-program channel** 与我们进行探讨，Issue 指定的 Mentor 会尽可能在 24h 内予以回复。不过，在提出问题之前一定要确保你已经仔细阅读过题目内容并且已经完成了参考资料的学习哦~
```
Comment (gingerkidney) [inline challenge-Programs/performance-challenge-cn.md]:
```suggestion
在实现代码的过程中如果遇到问题，可以通过 **#performance-challenge-program** channel 与我们进行探讨，Issue 指定的 Mentor 会尽可能在 24h 内予以回复。不过，在提出问题之前一定要确保你已经仔细阅读过题目内容并且已经完成了参考资料的学习哦~
```

File: challenge-Programs/performance-challenge-cn.md (hunk 1 above)
Comment (Soline324) [reply challenge-Programs/performance-challenge-cn.md]: Thanks a lot.

**Comment by gingerkidney** [comment]
LGTM

**Pull request merged**